src/release/breaking-changes/page-transition-replaced-by-ZoomPageTransitionBuilder.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Page transitions replaced by ZoomPageTransitionsBuilder
+description: Using the latest page transition instead of the old one.
+---
+
+## Summary
+
+In order to keep libraries follow the latest OEM behavior,
+we've replaced the default page transition builders from
+`FadeUpwardsPageTransitionsBuilder` to
+`ZoomPageTransitionsBuilder` on all platforms excluding iOS and macOS.
```

Comment:
```suggestion
In order to ensure that libraries follow the latest OEM behavior,
the default page transition builders now use
`ZoomPageTransitionsBuilder` on all platforms (excluding iOS and macOS)
instead of `FadeUpwardsPageTransitionsBuilder`.
```